src/libsync/propagateremotedelete.cpp

```diff
@@ -13,67 +13,20 @@
  */
 
 #include "propagateremotedelete.h"
+#include "abstractpropagateremotedeleteencrypted.h"
```

Comment:
I'd say this include is unneeded since it'll be brought by the other two PropagateRemoteDeleteEncrypted classes.